### PR TITLE
Education report cleanup

### PR DESCRIPTION
**Pull request by dmccurdybah**
## Description of change
Remove the ST Louis RPO from the education FYTD report starting with the 2021 fiscal year.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#13859

## Things to know about this PR
- Change is behind the `education_reports_cleanup` feature flag
- Tested locally and QA reviewed

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]

**Review by delli-gatti-michael — APPROVED** [review]
Looks good to me

File: spec/jobs/education_form/create_daily_fiscal_year_to_date_report_spec.rb [hunk 1]
```diff
@@ -78,6 +79,8 @@ def get_education_form_fixture(filename)
       create(:education_benefits_submission, form_type: '0993', created_at: date, region: :western)
       create(:education_benefits_submission, form_type: '0994',
                                              created_at: date, region: :eastern, vettec: true, chapter33: false)
+
+      # create(:education_benefits_submission, form_type: 'FAKE', created_at: date)
```
Comment (thilton-oddball) [inline spec/jobs/education_form/create_daily_fiscal_year_to_date_report_spec.rb:83]: Just double checking to see if you meant to include this commented out line in there.

**Review by thilton-oddball — APPROVED** [review]
One comment to review but otherwise looks good.

File: spec/jobs/education_form/create_daily_fiscal_year_to_date_report_spec.rb (hunk 1 above)
Comment (dmccurdybah) [reply spec/jobs/education_form/create_daily_fiscal_year_to_date_report_spec.rb:83]: Thanks, I did not!

**Pull request merged**